node/bundler.ts

```diff
@@ -106,36 +109,40 @@ const bundle = async (
   // rename the bundles to their permanent names.
   await createFinalBundles([functionBundle], distDirectory, buildID)
 
-  // Retrieving a configuration object for each function.
-  const functionsConfig = await Promise.all(
-    functions.map((func) => getFunctionConfig(func, importMap, deno, logger, featureFlags)),
-  )
-
   // Creating a hash of function names to configuration objects.
-  const functionsWithConfig = functions.reduce(
-    (acc, func, index) => ({ ...acc, [func.name]: functionsConfig[index] }),
-    {} as Record<string, FunctionConfig>,
-  )
+  const internalFunctionsWithConfig: Record<string, FunctionConfig> = {}
+  for (const func of internalFunctions) {
+    internalFunctionsWithConfig[func.name] = await getFunctionConfig(func, importMap, deno, logger, featureFlags)
+  }
+
+  const userFunctionsWithConfig: Record<string, FunctionConfig> = {}
+  for (const func of userFunctions) {
+    userFunctionsWithConfig[func.name] = await getFunctionConfig(func, importMap, deno, logger, featureFlags)
+  }
```

Comment:
This removes the parallelism of `getFunctionConfig` but is for now much easier to read, we can try to make `getFunctionConfig`s run in parallel later again.

Reply:
`getFunctionConfig` means spinning up the Deno CLI, running a function, writing the result to a file. This is a non-trivial operation. Have we considered the impact of losing parallelisation here?

Reply:
Okay it is parallel again